packages/x-charts/src/internals/domUtils.ts

```diff
@@ -152,3 +152,9 @@ export const getStringSize = (text: string | number, style: React.CSSProperties
     return { width: 0, height: 0 };
   }
 };
+
+// eslint-disable-next-line @typescript-eslint/naming-convention
+export function unstable_cleanupDOM() {
+  const measurementSpan = document.getElementById(MEASUREMENT_SPAN_ID);
+  measurementSpan?.remove();
+}
```

Comment:
Cant we empty the innertext or remove the element before returning the results of `getStringSize` instead?

Reply:
That seems to have broken the pipeline 😅

Reply:
No, that was just some outrage on GitHub ;)